### PR TITLE
Governance: Use optional for council mint

### PR DESCRIPTION
**Pull request by SebastianBor**

### CODE REVIEW

File: packages/proposals/src/actions/registerProgramGovernance.ts [hunk 1]
```diff
@@ -111,11 +112,15 @@ export const registerProgramGovernance = async (
     );
   }
 
+  const council_mint_seed = uninitializedTimelockConfig.councilMint
```
Comment (bhgames) [inline packages/proposals/src/actions/registerProgramGovernance.ts:115]: camel case in JS land, please :)

Comment (SebastianBor) [reply packages/proposals/src/actions/registerProgramGovernance.ts]: He, he, I've been brainwashed by Rust :), fixed

File: packages/proposals/src/models/initTimelockConfig.ts [hunk 2]
```diff
@@ -13,19 +13,19 @@ import * as Layout from '../utils/layout';
 ///           program account key, governance mint key, council mint key, timelock program account key.
 ///   1. `[]` Program account that this config uses
 ///   2. `[]` Governance mint that this config uses
-///   3. `[]` Council mint that this config uses [Optional] [Pass in 0s otherwise]
+///   3. `[]` Council mint that this config uses [Optional]
```
Comment (bhgames) [inline packages/proposals/src/models/initTimelockConfig.ts:16]: I noticed in your other PR (#7) which I merged that you haven't re-arranged Council Mint to be the last argument but here you have. You need to reconcile that.

Comment (SebastianBor) [reply packages/proposals/src/models/initTimelockConfig.ts:16]: I did rearrange it [#7](https://github.com/bhgames/solana-program-library/pull/7/filesl)

**Comment by bhgames** [comment]
Approved

**Review by bhgames — APPROVED** [review]

**Pull request merged**